frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/controlData/ControlDataKubeVirt.js

```diff
@@ -44,16 +44,20 @@ const operatorAlert = (localCluster, t) => {
   )
 }
 
-export const setKubeVirtSecrets = (control) => {
+export const onChangeKubeVirtConnection = (control) => {
   const { active, availableMap = {} } = control
   const replacements = get(availableMap[active], 'replacements')
+  const isEncoded = replacements?.encoded && replacements?.encoded === true
   const activePullSecret = replacements?.pullSecret ?? ''
   const activeSSHKey = replacements?.['ssh-publickey'] ?? ''
-  const isEncoded = replacements?.encoded && replacements?.encoded === true
+  const activeExternalInfraKubeconfig = replacements?.externalInfraKubeconfig ?? ''
+  const activeExternalInfraNamespace = replacements?.externalInfraNamespace ?? ''
 
   if (active && !isEncoded && activePullSecret !== '') {
     control.availableMap[active] = {
       replacements: {
+        externalInfraKubeconfig: Buffer.from(activeExternalInfraKubeconfig, 'ascii').toString('base64'),
+        externalInfraNamespace: activeExternalInfraNamespace,
```

Comment:
I think we need to investigate here what `encoded: true` does and if there's any potential problem with namespace not being encoded.

Reply:
sure 👍

Reply:
Actually, this `encoded` field is added to the `replacements` map, but it is not used by temptifly... just locally in this code. It was introduced in this PR: https://github.com/stolostron/console/pull/3330

Since you haven't changed anything about how it is used, I think it should be fine. Resolving comment.